prosody/rootfs/defaults/conf.d/jitsi-meet.cfg.lua

```diff
@@ -134,7 +135,9 @@ VirtualHost "jigasi.meet.jitsi"
       "muc_password_check";
     }
     authentication = "token"
+    signature_algorithm = "{{ .Env.JWT_SIGN_TYPE }}"
```

Comment:
Use the variable you have declared, not .Env

Reply:
Thanks, fixed it!